assets/js/modules/ads/pax/services.js

```diff
@@ -16,11 +16,35 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
 /**
  * Internal dependencies
  */
 import { CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
 
+const restFetchWpPages = async () => {
+	try {
+		const wpPages = await apiFetch( {
+			path: '/wp/v2/pages?per_page=100',
+		} ).then( ( pages ) =>
```

Comment:
Let's avoid mixing await with then/catch

Reply:
This has been addressed thanks @aaemnnosttv